tests/centosci/sink-clustered-deployment.sh

```diff
@@ -34,7 +34,7 @@ IMG="${CI_IMG_OP}" make test
 
 # Run integration tests
 SMBOP_TEST_CLUSTERED=1 SMBOP_TEST_EXPECT_MANAGER_IMG="${CI_IMG_OP}" \
-	./tests/test.sh
+	./tests/test.sh || (res=$? ; ./tests/post-test-info.sh; exit "$res")
```

Comment:
How about the following change?

```suggestion
	./tests/test.sh || (./tests/post-test-info.sh && exit 1)
```

Reply:
It's fine, but I figured I'd go the extra mile and preserve the original error code. Please confirm if you'd like the "exit 1" version and I'll make the change.

Reply:
done